app/src/main/java/de/tum/in/tumcampusapp/activities/SurveyActivity.java

```diff
@@ -204,35 +231,35 @@ private void zoomOutanimation(View v) {
         parentView.startAnimation(zoomOut);
         zoomOut.setAnimationListener(new Animation.AnimationListener() {
             @Override
```

Comment:
Why do you need to set the AnimationListener here? Just remove it, as you are not animating anything? Or add a comment on why it is needed.